samples/v1/speech_transcribe_async.js

```diff
@@ -0,0 +1,82 @@
+// Copyright 2019 Google LLC
```

Comment:
For the binaries above - these can really inflate the size of the git index over time, since they can't be diffed.  Have we explored storing these assets in GCS and using references to them?